tool/internal_pkg/tpl/server.go

```diff
@@ -49,4 +49,11 @@ func NewServer(handler {{call .ServiceTypeName}}, opts ...server.Option) server.
     return svr
 }
 {{template "@server.go-EOF" .}}
+
```

Comment:
1. It should be simplified as just:
```
return svr.RegisterService(serviceInfo(), handler, opts...)
```

2. Why is the need of a function wrapping a simple like which has little difference?

Reply:
1. You are right.
2. We don't want to expose serviceInfo to users because it's a pointer.